include/swift/AST/TypeRepr.h

```diff
@@ -613,38 +613,38 @@ class NamedTypeRepr : public TypeRepr {
 
 /// \brief A protocol composite type.
 /// \code
-///   protocol<Foo, Bar>
+///   Foo & Bar
 /// \endcode
 class ProtocolCompositionTypeRepr : public TypeRepr {
   ArrayRef<IdentTypeRepr *> Protocols;
-  SourceLoc ProtocolLoc;
-  SourceRange AngleBrackets;
+  SourceLoc FirstTypeLoc;
+  SourceRange CompositionRange;
```

Comment:
Ah, I meant `getLocImpl`, sorry. I think `getLocImpl` should point to the (first) `&` rather than the first protocol.